Calendar calendar state context set value can not set null value (#6030)
adobe/react-spectrum: packages/react-aria-components/docs/Calendar.mdx

</Calendar>
```

#### Reset value

> This example shows the `Footer` component, which is placed inside the `Calendar`. In this component, we use `CalendarStateContext` to get the `setValue` method from the `CalendarState` object. Next, by clicking on the button, we set the `setValue` to `null` to reset the `Calendar` `value`.

Reviewer: ```suggestion
This example shows a custom `Footer` component that can be placed inside the `Calendar.` In this component, we use `CalendarStateContext` to get the `setValue` method from the `CalendarState` object. Next, by clicking on the button, we set the `setValue` to `null` to reset the `Calendar` `value`.
```
